### ISSUE TITLE
Improve scraper to provide results for image, videos and file-types

### ISSUE THREAD

**Issue by mariobehling** [fossasia/query-server#58]
Image and video results don't have any endpoint in API of query server currently, which specifically takes file-type as input. Please implement it to allow us to include a query service for images and videos as well.

**Comment by shashank-sharma** [comment]
I would like to work on this issue.

**Comment by bhaveshAn** [comment]
@mariobehling Is this issue still active. Please let me know  I am willing to work on it. 😄

**Comment by mariobehling** [comment]
Right now, we only get general search results from search engines. Next, the goal is to get search results that are limited to e.g. video results - just like in the Google video tab.

**Comment by himanshukumar660** [comment]
Hello @mariobehling, I would like to work on this issue.

**Comment by raju249** [comment]
Any update on this issue @himanshukumar660 ?

**Comment by himanshukumar660** [comment]
@raju249 Yes, I'm working on it. Maybe you could give me sometime as i am having trouble building it. If i am not able to successfully proceed on it within some days, i will inform you.

**Comment by raju249** [comment]
Hey no problem. Take you time. 😊
I just wanted that it's not left out ☺️
Go ahead

**Comment by himanshukumar660** [comment]
Thanks



‌

On Mon, Nov 27, 2017 at 10:00 AM, Rajendra kadam <notifications@github.com>
wrote:

> Hey no problem. Take you time. 😊
> I just wanted that it's not left out ☺️
> Go ahead
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://mailtrack.io/trace/link/6b86d5b379ed240a9dd25762a0fd82e466a563b8?url=https%3A%2F%2Fgithub.com%2Ffossasia%2Fquery-server%2Fissues%2F58%23issuecomment-347076274&userId=1839408&signature=602881751b2194d8>,
> or mute the thread
> <https://mailtrack.io/trace/link/ebc28e18077f05d9c2465d92b73e9b51da35c7a2?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAQFgzZAKQsPuHZw0jQEShR4TbDPkuo3hks5s6jrWgaJpZM4Nzs32&userId=1839408&signature=f150fad80e34f2c9>
> .
>



-- 
Himanshu Kumar
B. Tech (Penultimate year)
Computer Science and Engineering
Ph. no. - (+91) 9476001014
National Institute of Technology, Durgapur
Mahatma Gandhi Avenue,
Durgapur, West Bengal- 713209
India

**Comment by vaibhavsingh97** [comment]
@himanshukumar660 Please check #320 and #321. It's already claimed, please ask the claimers if they are working on issue or not. Since I claimed #320 so will send PR shortly. Sorry for the delay

**Comment by raju249** [comment]
Shall we close this since the features needed here are taken care of in #320 and #321 .
@vaibhavsingh97 ?

/cc @mariobehling @bhaveshAn

**Comment by bhaveshAn** [comment]
Please see it has a single issue which has been split up by #320 and #321

**Comment by dilraj45** [comment]
Is this issue still open? I would really like to work on this.
I can start with developing APIs for scraping images and video results obtained from various search engines.

**Comment by vaibhavsingh97** [comment]
@dilraj45 you can look at #320 and #321 for the status. As of now both are occupied, you can look over new issues
